src/main/java/duckie/task/TaskList.java

```diff
@@ -0,0 +1,106 @@
+package duckie.task;
+
+import java.util.ArrayList;
+
+import duckie.Ui;
+import duckie.exception.DuckieException;
+import duckie.exception.DuckieNoListException;
+
+/**
+ * Contains all the saved tasks in a List form
+ */
+public class TaskList {
```

Comment:
Could consider if TaskList should be placed inside the task package or along side the duckie class